cocotb/triggers.py

```diff
@@ -721,10 +701,6 @@ class First(_AggregateWaitable):
 
     Returns the result of the trigger that fired.
 
-    As a shorthand, ``t = yield [a, b]`` can be used instead of
-    ``t = yield First(a, b)``. Note that this shorthand is not available when
-    using :keyword:`await`.
```

Comment:
I think we need to keep this to help explain to existing users that `First` is just another name for the `yield [...]` syntax they're used to.

However, we could move it down to the bottom of the docstring, and remove the word "shorthand" to indicate that it isn't worth switching to. Perhaps:
```
.. note::
    When using the old-style :keyword:`yield`-based coroutines, ``t = yield [a, b]`` was another spelling
    of ``t = yield First(a, b)``. This spelling is no longer available when using :keyword:`await`-based
    coroutines.